libnavigation-ui/src/main/java/com/mapbox/navigation/ui/map/NavigationMapboxMap.java

```diff
@@ -712,10 +720,12 @@ private void initializeLocationComponent(MapView mapView, MapboxMap map) {
     LocationComponentOptions options = LocationComponentOptions.createFromAttributes(context, locationLayerStyleRes);
     LocationComponentActivationOptions activationOptions = LocationComponentActivationOptions.builder(context, style)
             .locationComponentOptions(options)
-            .useDefaultLocationEngine(false)
+            .useDefaultLocationEngine(true)
```

Comment:
The component shouldn't use the built-in engine. It should display location updates received from Nav Core.